bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
I was mentioned by Jiri Simsa that: '"bin/tachyon-mount.sh SudoMount
workers" does not set the $bin variable, resulting in incorrect behavior'
so I'm just giving a better assignment to those variables



Abraços,


Carlos Anibal Amaral de Carvalho
móvel: (11) 95203-1505
email: caadecarvalho@gmail.com
Facebook <https://www.facebook.com/camaralc> - LinkedIn
<http://br.linkedin.com/pub/carlos-anibal-amaral-de-carvalho/24/628/740/> -
Blog <http://caadecarvalho.wordpress.com>

2016-01-18 21:06 GMT-02:00 chang cheng <notifications@github.com>:

> In bin/tachyon-workers.sh
> <https://github.com/amplab/tachyon/pull/2501#discussion_r50054308>:
>
> > @@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
> >  fi
> >
> >  bin=`cd "$( dirname "$0" )"; pwd`
> > -DEFAULT_LIBEXEC_DIR="$bin"/../libexec
>
> I think the original behavior is fine? What's the problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amplab/tachyon/pull/2501/files#r50054308>.
>